src/conversions.jl

```diff
@@ -434,6 +434,46 @@ julia> AstroTime.Epochs.ut1tt(ut1.jd1, ut1.jd2, AstroTime.Epochs.deltat(ut1))
      date, date1
 end
 
+@inline function jd2cal(jd1, jd2)
+    dj = jd1 + jd2
+    if dj < JD_MIN || dj > JD_MAX
+        error("Date is not acceptable")
```

Comment:
You should use plain `error` only in scripts. Use more explicit exception types in library code. Here `throw(ArgumentError("..."))` would be appropriate.

I would also like a more descriptive error message, e.g. "Julian date is outside of the representable range." or something like that.